appinventor/appengine/src/com/google/appinventor/client/editor/simple/SimpleVisibleComponentsPanel.java

```diff
@@ -32,7 +36,8 @@
   // UI elements
   private final VerticalPanel phoneScreen;
   private final CheckBox checkboxShowHiddenComponents;
-  private final CheckBox checkboxPhoneTablet; // A CheckBox for Phone/Tablet preview sizes
+  private final ListBox listboxPhoneTablet; // A CheckBox for Phone/Tablet preview sizes
```

Comment:
Update the comment here because it's a listbox, not a checkbox.

Reply:
Done